llama_stack/providers/utils/inference/openai_compat.py

```diff
@@ -174,6 +174,7 @@ def process_chat_completion_response(
 ) -> ChatCompletionResponse:
     choice = response.choices[0]
 
+    # TODO: This does not work well with tool calls for vLLM remote provider
```

Comment:
can you create an issue and point here to make sure we track it to completion?

Reply:
Added https://github.com/meta-llama/llama-stack/issues/1058